index.html

```diff
@@ -1,20 +1,221 @@
+
+        <section class="about-me row md-col-12 lg-col-12 col-12">
+          <aside class="about-me-aside md-col-4 lg-col-4 col-12">
+            <h1>Feature</h1>
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it to make a type specimen book. It has
+              survived not only five centuries, but also the leap into
+              electronic typesetting, remaining essentially unchanged. It was
+              popularised in the 1960s with the release of Letraset sheets
+              containing Lorem Ipsum passages, and more recently with desktop
+              publishing software like Aldus PageMaker including versions of
+              Lorem Ipsum.
+            </p>
+          </aside>
+
+          <img
+            src="https://images.unsplash.com/photo-1579889328266-6c54bc8d808b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=675&q=80"
+            class="about-me-photo-first-row sm-col-12 md-col-4 lg-col-4 col-12"
+          />
+
+          <article class="about-me-aside md-col-4 lg-col-4 col-12">
+            <h1>News</h1>
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it to make a type specimen book. It has
+              survived not only five centuries, but also the leap into
+              electronic typesetting, remaining essentially unchanged. It was
+              popularised in the 1960s with the release of Letraset sheets
+              containing Lorem Ipsum passages, and more recently with desktop
+              publishing software like Aldus PageMaker including versions of
+              Lorem Ipsum.
+            </p>
+          </article>
+        </section>
+        <section class="container">
+          <h1 class="title">projects</h1>
+          <article class="row">
+            <img
+              src="https://images.unsplash.com/photo-1524578339099-e987812b11b0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=80"
+              class="about-me-photo sm-col-4 md-col-4 lg-col-4 col-6"
+            />
+            <img
+              src="https://images.unsplash.com/photo-1523357585206-175e971f2ad9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
+              class="about-me-photo sm-col-4 md-col-4 lg-col-4 col-6"
+            />
+            <img
+              src="https://images.unsplash.com/photo-1534072700421-3100bd80c44d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60"
+              class="about-me-photo sm-col-4 md-col-4 lg-col-4 col-12"
```

Comment:
Don't use the `col-...` classes directly on images. Create a div for them, and wrap the image inside the div:
```
<div class="col-12 md-col-4">
    <img class="about-me-photo" src="" alt="">
</div>
```